src/app/lib/utilities/getRemoteConfig/index.test.js

```diff
@@ -0,0 +1,85 @@
+import fetchMock from 'fetch-mock';
+
+const mockUrl =
+  'https://mock-config-endpoint?application=simorgh&service=mundo&__amp_source_origin=http://localhost';
+const mockResponse = { ads: { enabled: true } };
```

Comment:
Can we add `testToggle` here instead of `ads` (and in other locations within the PR)

Reply:
Old habits, fixed this. I've updated it elsewhere.

`withContexts` still has 
```js
    remoteConfig: shape({
      toggles: shape({
        ads: shape({
          enabled: bool.isRequired,
        }),
      }),
    }),
```

But you have that annoying scenario where you aren't allowed to use object type, but also don't want to specify the shape. I guess `object.isRequired` and disabled lint on that line?

Reply:
Happy to leave it for now